### DISCUSSION TITLE
Welcome to quantumpi-nexus Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/quantumpi-nexus#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Emma0945** [comment]
We are developping pi community.

**Comment by thelobixoo** [comment]
Puede confirmar ese precio de 314k usd ?

**Comment by Anggono9** [comment]
Tanggal berapa open mainnet?

**Comment by Emma0945** [comment]
Hi.I want to see your new phone mobile
